activerecord/lib/active_record/transactions.rb

```diff
@@ -289,6 +298,7 @@ def raise_in_transactional_callbacks=(value)
       def set_options_for_callbacks!(args, enforced_options = {})
         options = args.extract_options!.merge!(enforced_options)
         args << options
+        options[:prepend] = true unless ActiveRecord::Base.retain_reversed_transactional_callbacks_order
```

Comment:
`self.retain_reversed_transactional_callbacks_order` so you can even disable this value in you specific class.